src/compiler/scanner.ts

```diff
@@ -1967,7 +1967,18 @@ export function createScanner(
                         }
                         return pos += 2, token = SyntaxKind.ExclamationEqualsToken;
                     }
+                    const hasLeadingWhitespace = charCodeChecked(pos - 1) === CharacterCodes.space;

```

Comment:
This doesn't seem like the right check; the character could be a tab or other character. I think we have a helper for this sort of check.

Reply:
Thanks; you'll probably want a test with a tab in it just to verify.